src/main/java/uk/gov/hmcts/reform/em/test/idam/OpenIdConfiguration.java

```diff
@@ -7,20 +7,20 @@
 public class OpenIdConfiguration {
 
     private String clientId;
-    private String client_secret;
+    private String clientSecret;
     private String redirectUri;
-    private String grant_type;
+    private String grantType;
     private String scope;
 
     public OpenIdConfiguration(@Value("${idam.client.id:}") String clientId,
-                               @Value("${idam.client.secret:}") String client_secret,
+                               @Value("${idam.client.secret:}") String clientSecret,
                                @Value("${idam.client.redirect_uri:}") String redirectUri,
                                @Value("${idam.client.scope:}") String scope,
-                               @Value("${idam.client.grant_type:}") String grant_type) {
+                               @Value("${idam.client.grantType:}") String grantType) {
```

Comment:
Could you please just revert this one attribute change? As this will break all the EM API's consuming this library?
https://github.com/hmcts/em-stitching-api/blob/master/src/aat/resources/application.yml#L44C5-L44C15
Also, I have checked in other Reform API's and seems all are following current naming convention of grant_type.

Reply:
Okay It does come up as an error for the check style